migration-from-4x.md

```diff
@@ -378,11 +386,17 @@ editor.ui.registry.addButton('customDateButton', {
   icon: 'insert-time',
   tooltip: 'Insert Current Date',
   disabled: true,
-  onAction: (_) => editor.insertContent(toTimeHtml(new Date())),
-  onSetup: (buttonApi) => {
-    const editorEventCallback = (eventApi) => buttonApi.setDisabled(eventApi.element.nodeName.toLowerCase() === 'time');
+  onAction: function (_) {
```

Comment:
I'd suggest removing the `_` here, as it's more of an internal convention, than something someone else would understand. eg:
```
onAction: function () {
```

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d